docs/05-operations/01-duration-terms.adoc

```diff
@@ -14,7 +14,7 @@ Monitoring, Operations, Logging, Tracing, Metrics, Alerting.
 |===
 
 === Terms and Principles
-Term 1, Term 2, Term 3
+Monitoring, Operations, Logging, Tracing, Metrics.
```

Comment:
German text also has "Alerting". Not relevant in English, or not relevant in general?